test/e2e/apimachinery/garbage_collector.go

```diff
@@ -328,15 +330,23 @@ func newCronJob(name, schedule string) *batchv1beta1.CronJob {
 	}
 }
 
+// getUniqLabel returns a UniqLabel based on labeLkey and labelvalue.
+func getUniqLabel(labelkey, labelvalue string) map[string]string {
+	atomic.AddInt64(&lablecount, 1)
```

Comment:
you need to use the return value from this add as your value down below.  Otherwise your writes and readers are racing.

Reply:
Done

Reply:
cc @sttts @deads2k